docs/core-concepts/index.md

```diff
@@ -0,0 +1,31 @@
+# Core Concepts
+
+These are some of the core concepts that you should be familiar with before going over the [Getting Started Guide](get-started/index.md).
+
+
+1. **Git Source**
```

Comment:
Instead of making this a list, I would use titles (e.g. `## Git Source`) so that it possible to link to them using an anchor.

Reply:
Done!